src/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -92,7 +92,6 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->addDefaultsIfNotSet()
                     ->children()
                         ->variableNode('serialize_payload_fields')->defaultValue([])->info('Set to null to serialize all payload fields when a validation error is thrown, or set the fields you want to include explicitly.')->end()
-                        ->booleanNode('query_parameter_validation')->defaultValue(true)->end()
```

Comment:
I forgot to deprecate this therefore I think that it should be first deprecated in 4.2 then removed in 5.0.